packages/components/Checkbox/SPEC.md

```diff
@@ -1,5 +1,7 @@
 # Checkbox
 
+**Not available on MacOS** - Please use Checkbox V0 under the deprecated folder until further notice.
+
```

Comment:
Rather than `under the deprecated folder` , maybe we can give the import path? Since people reading this probably don't care about the repo structure but what code to write?

Reply:
Done